DigitalLearningSolutions.Web/Views/LearningPortal/Current/Current.cshtml

```diff
@@ -55,9 +55,9 @@
         }
       </div>
     }
-    @if (Model.TotalPages > 1) {
-      <partial name="SearchablePage/_Pagination" model="Model" />
-    }
+
+    <partial name="SearchablePage/_Pagination" model="Model" />
+
 

```

Comment:
Another minor formatting thing, we should never have multiple blank lines. There are several files here where the spacing from below the if block has been kept, but so has the blank line that results from deleting the closing bracket. The formatter won't catch these for cshtml files, so they'll need a quick manual tidy up.